### DISCUSSION TITLE
Part3 Ch2_03 pod-pod과 컨테이너들간의 통신

### DISCUSSION THREAD

**Discussion by nimo1996** [yoonjeong-kwon/fastcampus-kubernetes#20] [Q&A] [answered]
파드 상태
![image](https://user-images.githubusercontent.com/72293595/214505780-26987b4d-18a7-4d25-9b73-251552d2d420.png)

blue-app 컨테이너 -> red-app 컨테이너 /rose, /hello 요청 실행 상태
![image](https://user-images.githubusercontent.com/72293595/214506230-1f87b286-3cff-4de5-945b-da087c1f2909.png)

사용중인 포트 & 방화벽 상태
![image](https://user-images.githubusercontent.com/72293595/214507496-7083912c-b1a3-43ea-ba74-327027fe086e.png)

현재 상태입니다. 

이상태에서 브라우저에서 localhost:8080 으로 접속하려고 할때 사이트에 연결할 수 없다는 메세지만 나오고 있습니다.

혹시 더 확인해봐야하는 사항이 있을까요?

**Comment by yoonjeong-kwon** [comment, marked as answer]
@nimo1996 님, 안녕하세요.
아래 명령어를 이용하면 로컬 포트를 쿠버네티스 클러스터에서 실행중인 파드의 포트로 포트포워딩을 실행할 수 있습니다.
`kubectl port-forward <pod-name> <host-port>:<container-port>`
이 명령어는 chapter02-02, 03 실습에서 다루고 있습니다. 이것을 실행보시기 바랍니다.
감사합니다.
